test/integration/targets/pids/tasks/main.yml

```diff
@@ -3,18 +3,18 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 - name: "Installing the psutil module"
   pip:
-    name: psutil
+    name: psutil==5.6.7
 
-- name: "Checking the empty result" 
-  pids: 
```

Comment:
![space](https://media.giphy.com/media/3oGRFkmoqoui9nzL2g/source.gif)

Reply:
Oops! I didn't mean to add that to the PR. I have my editor set to nuke trailing spaces so I sometimes end up with things like that.

Sorry, late on a Friday. 🤯